t/00.uri_version.t

```diff
@@ -0,0 +1,13 @@
+use strict;
+use warnings;
+
+use Test::More;
+
+use URI ();
+
+my $expected_version = '5.12';  #MAINT: Update for each new version.
```

Comment:
I'm not sure we need to handle this case in the PR. That is, I don't think we need to guard against someone's broken Perl installation. Also, I'd like to avoid adding any manual tasks to the release process.

Reply:
To be clear, I'm not sure if this PR is something we should merge. I realize it solves a problem, but this may not be our problem to solve.

Reply:
It would remind me to use `prove -l` instead of `prove` ;-)

Regarding the manual workload. The first invocation of `prove -l` will remind one to update the version info. Not much impact and something that will not be overseen. 
However, I can live with the decision not to merge it. Work put into it is negligible. 

But I do not think that this is a bug in `URI` module, so the `bug`-tag should be removed.

Reply:
Ok, I understand. As said before, I do not care too much about this PR. It's an offer that is cool to be rejected.

BTW, if automation is wanted: 
I could offer an inode based method (basically inode( somewhere/t/../lib/URI.pm ) is the same as inode( $INC{'URI.pm'} )) which requires no maintenance. Don't know how portable this is - the POD says, it would work
on any OS as long as string comparison is used.